doc/user-manual/tools/literate-programming.rst

```diff
@@ -161,6 +161,12 @@ Literate Forester
 Files ending in :file:`.lagda.tree` are interpreted as literate
 Forester_ files. Literate forester use ```\agda{...}``` for code blocks.
 
+* You will need the postprocessor agda-tree_ to convert ``html/*.tree`` to a valid forester tree.
+
+  * ``agda --html --html-highlight=code example.lagda.tree`` will produce the file ``html/example.tree``.
+  * Run ``agda-tree build`` at where ``html/`` is located, this will produce subdirectory ``trees/`` there.
+  * Add ``trees/`` to ``forest.toml``.
```

Comment:
These instructions are still incomplete, I think.
I do not get any highlighted Agda code this way.
Something has to be done with `Agda.css` also maybe?
Or where does the coloring information come from?

Reply:
Highlight is came from `Agda.css`. I think what missing here is "Add generated html/ directory to be assets in forest.toml"

Reply:
updated.

Reply:
![image](https://github.com/user-attachments/assets/f5c8bfb7-8fbb-420d-abda-f4ec9163aadb)

Here is the pure environment I setup.